CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ All notable changes to the "vscode-camelk" extension will be documented in this
 - Adapt Didact tutorial registration to VS Code Didact 0.4.0 updated API
 - Update default runtime version to v1.5.0
 - `resource` has been replaced by `resources` to allow providing several of them in Camel K VS Code Tasks
+- Actions related to starting an integration with resource, configmap or secret now require kamel 1.5.0 community or 1.4.0 Red Hat.
```

Comment:
Couple of minor suggestions:
* missing a final comma in this comma-delimited list (https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/)
* and remove the period at the end to match the other entries in this change log

With these minor adjustments, the line becomes:
"Actions related to starting an integration with resource, configmap, or secret now require kamel 1.5.0 community or 1.4.0 Red Hat"